modules/test/playwright/pages/account-admin-web/DataTablePage.ts

```diff
@@ -143,7 +143,7 @@ export class DataTablePage {
 		this.rowActions = async (value, colIndex = 1, strictEqual = true) => {
 			const row = await this.row(colIndex, value, strictEqual);
 
-			if (row && row.column) {
+			if (row && row.row) {
```

Comment:
mmm ... it should be ok but probably we have to run the full playwright suite

Reply:
yes, you're right but I don't know if "later" someone is using the .column expecting it's always there.
But I think that if we have the .row we have also the .column.
Btw if you can have just a quick check on who is using it